### PR TITLE
Add column-aligner-ads

### LINKED ISSUE #20936: Add Extension column-aligner-ads
**Is your feature request related to a problem? Please describe.**
I've authored a small new extension for ADS that formats code to tab-align it with your selection.

**Describe the solution or feature you'd like**
Please add it to the extension gallery.

**Describe alternatives you've considered**
Local VSIX install, but then I can't share it with the cheering masses!

**Additional context**
I have a separate PR (sorry, new at this) to fix the fact that I included an icon in the registration.

### PR DESCRIPTION
**Pull request by richardtallent**
Added a new little extension I initially developed for VS Code, but it's equally (or even more) useful in ADS, so I made an ADS version as well. (I goofed up and accidentally added an icon for this due to copy/paste error, but can't edit it here since I used GitHub's direct-edit. Sent a different PR for fix that.)

This PR fixes #20936

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -4755,6 +4755,71 @@
...
+					"versions": [
+						{
+							"version": "1.0.0",
+							"lastUpdated": "10/23/2022",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
+									"source": "https://tallent.us/column-aligner-ads/column-aligner-ads-1.0.0.vsix"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4778]: Unfortunately we can't have direct VSIX links to non-MS extensions. You'll need to change the asset type to `Microsoft.SQLOps.DownloadPage`

Comment (Charles-Gagnon) [inline extensionsGallery.json:4778]: You can modify this in Github too - just go to https://github.com/richardtallent/azuredatastudio/blob/patch-1/extensionsGallery.json and edit the file there

**Review by Charles-Gagnon — CHANGES_REQUESTED** [review]

File: extensionsGallery.json [hunk 2]
```diff
@@ -4755,6 +4755,71 @@
...
+									"key": "Microsoft.VisualStudio.Code.Engine",
+									"value": ">=1.39.0"
+								},
+								{
+									"key": "Microsoft.AzDataEngine",
+									"value": "*"
+								},
+								{
+									"key": "Microsoft.VisualStudio.Services.Links.Source",
+									"value": "https://github.com/Azure/azure-cosmosdb-ads-extension"
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4816]: Fix link here

File: extensionsGallery.json (hunk 1 above)
Comment (richardtallent) [reply extensionsGallery.json:4778]: Changed the assetType, but are you saying I need a landing page rather than a link directly to the VSIX file as well?

**Review by richardtallent — COMMENTED** [review]
I think we're ready now?

File: extensionsGallery.json (hunk 1 above)
Comment (richardtallent) [reply extensionsGallery.json]: Ah! All done now.

File: extensionsGallery-insider.json [hunk 3]
```diff
@@ -4816,6 +4816,67 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+{
+					"extensionId": "94",
```
Comment (Charles-Gagnon) [inline extensionsGallery-insider.json:4821]: Update total count in this file too

File: extensionsGallery-insider.json [hunk 4]
```diff
@@ -4816,6 +4816,67 @@
...
+								},
+								{
+									"key": "Microsoft.VisualStudio.Services.Links.Source",
+									"value": "https://github.com/richardtallent/column-aligner-ads"
+								}
+							]
+						}
+					],
+					"statistics": [],
+					"flags": ""
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json:4879]: please bump the TotalCount below.

File: extensionsGallery.json [hunk 5]
```diff
@@ -4755,6 +4755,67 @@
...
+							"lastUpdated": "10/23/2022",
+							"assetUri": "",
+							"fallbackAssetUri": "fallbackAssetUri",
+							"files": [
+								{
+									"assetType": "Microsoft.SQLOps.DownloadPage",
+									"source": "https://github.com/richardtallent/column-aligner/releases/tag/v1.0.0"
+								},
+								{
+									"assetType": "Microsoft.VisualStudio.Services.Content.Details",
```
Comment (Charles-Gagnon) [inline extensionsGallery.json:4781]: You also need to add an icon file asset back in - even if it's just some simple one now you plan on replacing later.

**Comment by Charles-Gagnon** [comment]
@richardtallent There's still the two open comments to address before we can merge this in. Let us know if you have any questions!

File: extensionsGallery-insider.json (hunk 3 above)
Comment (richardtallent) [reply extensionsGallery-insider.json:4821]: updated. note that this has 1 more (80) than in the other file (79).

File: extensionsGallery.json (hunk 5 above)
Comment (richardtallent) [reply extensionsGallery.json:4785]: Done, in both files.

**Comment by richardtallent** [comment]
Count is updated, icon added...

File: extensionsGallery-insider.json (hunk 3 above)
Comment (alanrenmsft) [reply extensionsGallery-insider.json:4821]: yes, that is the mysql extension.

**Review by alanrenmsft — APPROVED** [review]

**Comment by alanrenmsft** [comment]
@richardtallent thanks! you should be able to see the new extension in ADS now.

**Pull request merged**